subprojects/jvm-services/src/test/groovy/org/gradle/jvm/toolchain/internal/JavaInstallationRegistryTest.groovy

```diff
@@ -23,10 +23,11 @@ import spock.lang.Specification
 
 class JavaInstallationRegistryTest extends Specification {
 
-    def tempFolder = createTempDir()
```

Comment:
Why move this to all methods?

Reply:
I think I needed this before my last changes, I can revert it.